File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/mastery/MasteryDetailResponse.java
package com.legue.axel.lolsummonertool.network.response.mastery;

import com.legue.axel.lolsummonertool.database.model.mastery.MasteryImage;

import java.util.List;

public class MasteryDetailResponse {

    public int id;
    public String name;
    public List<String> description;
    public MasteryImage image;
    public int ranks;
    public String prereq;

    public MasteryDetailResponse() {
    }

    public MasteryDetailResponse(int id, String name, List<String> description,
                                 MasteryImage image, int ranks, String prereq) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.image = image;
        this.ranks = ranks;
        this.prereq = prereq;
    }
}

File: app/src/main/java/com/legue/axel/lolsummonertool/network/response/mastery/MasteryDetailResponse.kt
package com.legue.axel.lolsummonertool.network.response.mastery

import com.legue.axel.lolsummonertool.database.model.mastery.MasteryImage

class MasteryDetailResponse(
        var id: Int,
        var name: String,
        var description: List<String>,
        var image: MasteryImage,
        var ranks: Int,
        var prereq: String
)